ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-master.yaml

```diff
@@ -55,8 +55,14 @@ tag_specification:
   namespace: ocp
 test_binary_build_commands: |
```

Comment:
qq: what does this actually do?

Reply:
This creates the `test_bin` image.  This image was previously just used for running the tox tests against openshift-ansible, executed in the `unit` job.  I'm now using this image for the step that creates the ssh-bastion.  I was considering switching to the `ansible` image instead now that I've got the bastion step working as well as ssh working on the `ansible` image.  The `ansible` image is used for running all the playbooks to provision the instances, prepare the repo files and run the scaleup.